### ISSUE TITLE
Angular 9 Ivy compiler issue

### ISSUE THREAD

**Issue by billdwhite** [vladborsh/ngxs-effects#2]
I'm excited to see that someone got around to adding this plugin; however, when I compile it in my Angular 9 Ivy project, I get this error:

```
    740     static ɵfac: ɵngcc0.ɵɵFactoryDef<Compiler, never>;
                                ~~~~~~~~~~~~
    node_modules/ngxs-effects/node_modules/@angular/core/core.d.ts:4730:25 - error TS2694: Namespace '"<omitted>node_modules/ngxs-effects/node_modules/@angular/core/src/r3_symbols"' has no exported member 'ɵɵFactoryDef'.

    4730     static ɵfac: ɵngcc0.ɵɵFactoryDef<PlatformRef, never>;
                                 ~~~~~~~~~~~~
    node_modules/ngxs-effects/node_modules/@angular/core/core.d.ts:6207:25 - error TS2694: Namespace '"node_modules/ngxs-effects/node_modules/@angular/core/src/r3_symbols"' has no exported member 'ɵɵFactoryDef'.

    6207     static ɵfac: ɵngcc0.ɵɵFactoryDef<SystemJsNgModuleLoader, [null, { optional: true; }]>;
                                 ~~~~~~~~~~~~
    node_modules/ngxs-effects/node_modules/@angular/core/core.d.ts:6397:25 - error TS2694: Namespace '"<omitted>node_modules/ngxs-effects/node_modules/@angular/core/src/r3_symbols"' has no exported member 'ɵɵFactoryDef'.

    6397     static ɵfac: ɵngcc0.ɵɵFactoryDef<Testability, never>;
                                 ~~~~~~~~~~~~
    node_modules/ngxs-effects/node_modules/@angular/core/core.d.ts:6442:25 - error TS2694: Namespace '"<omitted>node_modules/ngxs-effects/node_modules/@angular/core/src/r3_symbols"' has no exported member 'ɵɵFactoryDef'.

    6442     static ɵfac: ɵngcc0.ɵɵFactoryDef<TestabilityRegistry, never>;
                                 ~~~~~~~~~~~~
    node_modules/ngxs-effects/node_modules/@angular/core/core.d.ts:9407:25 - error TS2694: Namespace '"<omitted>node_modules/ngxs-effects/node_modules/@angular/core/src/r3_symbols"' has no exported member 'ɵɵFactoryDef'.

    9407     static ɵfac: ɵngcc0.ɵɵFactoryDef<ɵConsole, never>;
                                 ~~~~~~~~~~~~
    node_modules/ngxs-effects/dist/ngxs-effect/lib/effect-starter.service.d.ts:16:25 - error TS2694: Namespace '"<omitted>node_modules/ngxs-effects/node_modules/@angular/core/core"' has no exported member 'ɵɵFactoryDef'.

    16     static ɵfac: ɵngcc0.ɵɵFactoryDef<EffectStarterService, [null, { optional: true; }, { optional: true; }]>;
```

**Comment by vladborsh** [comment]
Hi, thanks a lot for reporting! Could you please provide a list of dependencies in your project (preferably part of package.json)?

**Comment by vladborsh** [comment]
fixed issue here 7305b4e55
please check out version 2.0.0
looking forward to your feedback!

**Comment by billdwhite** [comment]
Yes, I just verified that it works!  Thanks so much for the fix!
